source/lex.tex

```diff
@@ -915,8 +814,9 @@
 \pnum
 \indextext{name!length of}%
 \indextext{name}%
-The character classes XID_Start and XID_Continue
-are Derived Core Properties as described by \UAX{44}.
+\begin{note}
+The character properties XID_Start and XID_Continue are Derived Core Properties
```

Comment:
Changing "character classes" to "character properties" wasn't in the paper, but seems right; I added a note to the commit message.